contracts/VTokenInterfaces.sol

```diff
@@ -157,30 +157,45 @@ abstract contract VTokenInterface is VTokenStorage {
     /**
      * @notice Event emitted when interest is accrued
      */
-    event AccrueInterest(uint256 cashPrior, uint256 interestAccumulated, uint256 borrowIndex, uint256 totalBorrows);
+    event AccrueInterest(
+        uint256 indexed cashPrior,
+        uint256 indexed interestAccumulated,
+        uint256 indexed borrowIndex,
+        uint256 totalBorrows
+    );
 
     /**
      * @notice Event emitted when tokens are minted
      */
-    event Mint(address minter, uint256 mintAmount, uint256 mintTokens, uint256 accountBalance);
+    event Mint(address indexed minter, uint256 mintAmount, uint256 indexed mintTokens, uint256 indexed accountBalance);
```

Comment:
I wouldn't index uint parameters. This comment is applicable for the rest of events.